scripts/db_query_handler.py

```diff
@@ -0,0 +1,145 @@
+"""
+This module defines the DBQueryHandler class for handling database queries.
+"""
+
+import os
+import sqlite3
+
+from scripts.utils.config import get_key_str
+from scripts.utils.files import (is_path_existent, is_readable_existing_file,
+                                 is_writeable_file_directory)
+
+
+class DBQueryHandler:  # pylint: disable=too-few-public-methods
```

Comment:
I hate this pylint warning and I always disable it
you could use `.pylintrc` to disable it for all files from conf